engine-wasm/build-wasm.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
+set -euo pipefail
 
-cd "$PWD/src/main/resources/go"
-tinygo build -target=wasi -o ../compiled/cdc.wasm cdc.go
-cd "$OLDPWD"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+docker run --rm \
+    -v ${SCRIPT_DIR}/src/main/resources:/src \
+    -w /src tinygo/tinygo:0.34.0 bash \
```

Comment:
`-target=wasm-unknown` requires a more recent version of `tinygo` than what is packaged for most OSes, switched the build to use a container as it's easier to reproduce.